nfs-ganesha/build/build_deb

```diff
@@ -8,15 +8,29 @@ fi
 
 
 ## Install any setup-time deps
-# TODO -- upload install-deps.sh script that installs the setup time deps
-# to upstream repo or update this section to install these packages
+# TODO: add file to /etc/apt/sources.list.d/ to get repos for librgw-dev and libcephfs-dev and run sudo apt-get update
 
 # We need this for system, to build ganesha, and to enable FSAL_CEPH and 
 # FSAL_RGW in the .spec file when cmake command runs
-sudo apt-get install -y equivs cmake git libkrb5-dev bison flex librgw2-dev libcephfs-dev
+sudo apt-get -y install "
+cmake
+equivs
+libkrb5-dev
+bison
+flex
+librgw-dev
+libcephfs-dev
+"
+
+# Normalize variables across rpm/deb builds
+NORMAL_DISTRO="ubuntu"
+NORMAL_DISTRO_VERSION=$DISTRO
```

Comment:
@andrewschoen I seem to have mixed up what $DISTRO is here. $DISTRO should be assigned to NORMAL_DISTRO and NORMAL_DISTRO_VERSION should be "Xenial" since that is the only Ubuntu platform we are making it on. I'm not worried about hard coding any variable atm. If that sounds correct I'll go ahead and change it.